### PR TITLE
Update dark mode props on Selectable Card Icon

### PR DESCRIPTION
**Pull request by kellyeryan**
#### Runway Ticket URL

NUX-1285

#### Screens

Screen Shot 2020-08-06 at 4 58 14 PM

Screen Shot 2020-08-06 at 4 57 55 PM

#### Breaking Changes

Removed dark prop from both sides of kit

#### Checklist:

- [X] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [X] **SCREENSHOT** Please add a screen shot or two
- [X] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

**Comment by coleerikson** [comment]
Looks like you've lost the 'location' and 'Wifi' labels on the card icons

**Comment by bh247484** [comment]
> Looks like you've lost the 'location' and 'Wifi' labels on the card icons

**Comment by bh247484** [comment]
Sorry accidentally closed PR...

**Comment by bh247484** [comment]
Meant to point out that some of the Sass relies on our old dark class name convention.  [Follow this link to see code in question](https://github.com/powerhome/playbook/blob/v6.1.0/app/pb_kits/playbook/pb_selectable_icon/_selectable_icon.scss#L56-L84).

**Comment by kellyeryan** [comment]
Thanks! You were very helpful and I think I fixed it!

File: app/pb_kits/playbook/pb_selectable_card_icon/_selectable_card_icon.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@
       }
     }
 
-  &[class*=_dark] {
```
Comment (bh247484) [inline app/pb_kits/playbook/pb_selectable_card_icon/_selectable_card_icon.scss:35]: ✨✨ 🚀 ✨✨

File: app/pb_kits/playbook/pb_selectable_card_icon/_selectable_card_icon.scss [hunk 2]
```diff
@@ -32,7 +32,7 @@
...
+  &.dark {
```
Comment (bh247484) [inline app/pb_kits/playbook/pb_selectable_card_icon/_selectable_card_icon.scss:35]: ✨✨ 🚀 ✨✨

**Pull request merged**